bin/minify-assets.mjs

```diff
@@ -0,0 +1,219 @@
+  '@ember/component',
+  '@ember/controller',
+  '@ember/debug',
+  '@ember/deprecated-features',
+  '@ember/destroyable',
+  '@ember/enumerable',
+  '@ember/helper',
+  '@ember/instrumentation',
+  '@ember/modifier',
+  '@ember/object',
+  '@ember/owner',
+  '@ember/renderer',
+  '@ember/routing',
+  '@ember/runloop',
+  '@ember/service',
+  '@ember/template',
+  '@ember/template-compilation',
+  '@ember/template-compiler',
+  '@ember/template-factory',
+  '@ember/test',
+  '@ember/utils',
+  '@ember/version',
+  '@glimmer/destroyable',
+  '@glimmer/encoder',
+  '@glimmer/env',
+  '@glimmer/global-context',
+  '@glimmer/manager',
+  '@glimmer/node',
+  '@glimmer/opcode-compiler',
+  '@glimmer/owner',
+  '@glimmer/program',
+  '@glimmer/reference',
+  '@glimmer/runtime',
+  '@glimmer/tracking',
+  '@glimmer/util',
+  '@glimmer/validator',
+  '@glimmer/vm',
+  '@glimmer/wire-format',
+];
+import glob from 'glob';
+import nodeGzip from 'node-gzip';
+
+import { join } from 'node:path';
+import { readFileSync, writeFileSync } from 'node:fs';
+import { minify } from 'terser';
+import { transformSync } from '@babel/core';
+import * as brotli from 'brotli';
+import { partial } from 'filesize';
+const size = partial({ standard: 'jedec' });
+
+const root = join(process.cwd(), 'dist/packages');
+
+let min = {};
+let br = {};
+let gzip = {};
+
+let packageData = {
+  ember: [
+    /* pkg, min, gz, br */
+  ],
+  glimmer: [],
+};
+
+function totalMin(dataset) {
+  return dataset.reduce((a, b) => a + b[1], 0);
+}
+
+function totalGz(dataset) {
+  return dataset.reduce((a, b) => a + b[2], 0);
+}
+
+// function totalBr(dataset) {
+//   return dataset.reduce((a, b) => a + b[3], 0);
+// }
+
+import { buildMacros } from '@embroider/macros/babel';
+
+process.env.NODE_ENV = 'production';
+process.env.EMBER_ENV = 'production';
+
+const macros = buildMacros();
+let babelOptions = {
+  cwd: process.cwd(),
+  plugins: [...macros.babelMacros],
+};
+
+for (const pkg of packages) {
+  let jsFiles = glob.sync(`${root}/${pkg}/**/*.js`);
+
+  for (let file of jsFiles) {
+    let source = readFileSync(file, 'utf8');
+    let transformed = transformSync(source, {
+      ...babelOptions,
+      filename: file,
+    }).code;
+    let result = await minify(transformed, {
+      module: true,
+      mangle: false,
+      ecma: 2021,
+      compress: {
+        ecma: 2021,
+        passes: 3,
+        defaults: true,
+        keep_fargs: false,
+        keep_fnames: false,
+        /**
+         * Required for {{debugger}} to work
+         */
+        drop_debugger: false,
+      },
+    });
+
+    let minFileName = file + '.min';
+    writeFileSync(minFileName, result.code);
+
+    let compressed = brotli.compress(result.code, {
+      // mode: 1, // 0 = generic, 1 = text, 2 = font (WOFF2)
+      quality: 11, // 0 - 11
+      // lgwin: 22, // window size
+    });
+
+    // console.log(brotli.decompress(brotli.compress(result.code)).length, result.code.length);
```

Comment:
I feel like I'm missing something about brotli, because decompressing compressed code results in the same length of code.

but the numbers were just... _unreal_